src/com/serotonin/mango/web/dwr/MiscDwr.java

```diff
@@ -276,6 +276,7 @@ public void setHomeUrl(String url) {
 			url = url.substring(1);
 
 		// Save the result
+        Common.getUser().setHomeUrl(url);
```

Comment:
1. 		User user = Common.getUser();
		user.setHomeUrl(url);
		UserService userService = new UserService();
		userService.saveHomeUrl(user.getId(), url);

2. To check if the user is logged in, and start this method added line:		
Permissions.ensureValidUser();

3. Removed unused imports;